source/utilities.tex

```diff
@@ -15012,24 +15056,27 @@
 \item \ucode{20000} -- \ucode{2fffd}
 \item \ucode{30000} -- \ucode{3fffd}
 \end{itemize}
-The estimated width of other code points is 1.
+The field width of all other code points is 1.
 
 \pnum
-For a string in a non-Unicode encoding, the width of a string is unspecified.
+For a sequence of characters in a non-Unicode encoding,
```

Comment:
Are the following changes, which the paper for #6126 made for existing wording (https://github.com/cplusplus/draft/pull/6126/files#diff-afa9f66c3857a24dc4092685af2cbf9cbc44bbd35afa4b0eb26a14a324b0999dL15018), not required here?
```suggestion
For a sequence of characters in neither UTF-8, UTF-16, nor UTF-32,
```